manifests/params.pp

```diff
@@ -3,6 +3,7 @@
     String $keycloak_server_url = lookup('transmart_packer::keycloak_server_url', String, first, ''),
     String $keycloak_realm = lookup('transmart_packer::keycloak_realm', String, first, 'dev'),
     String $keycloak_client_id = lookup('transmart_packer::keycloak_client_id', String, first, 'transmart'),
+    String $keycloak_offline_token = lookup('transmart_packer::$keycloak_offline_token', String, first, ''),
```

Comment:
`transmart_packer::$keycloak_offline_token` should be `transmart_packer::keycloak_offline_token` (without `$`), I guess?

Reply:
you're right, thanks!